assignments/objects.js

```diff
@@ -19,26 +19,74 @@ const example = {
 
 // Write your intern objects here:
 
+const employee1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const employee2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@inypic.com",
+  "gender": "M",
+  "speak": "Hello, my name is Kennan"
```

Comment:
I believe they were looking for you to create a method that would return a greeting string:

"speak": () => {
return "Hello, my name is Kennan"
}

you could even interpolate so it says: return `Hello, my name is ${this.name}`